tests/codegen/sanitizer/cfi/dbg-location-on-cfi-blocks.rs

```diff
@@ -0,0 +1,31 @@
+// Verifies that the parent block's debug information are assigned to the inserted cfi block.
+
```

Comment:
FYI, something caused a blank line to be inserted every other line here.

Reply:
Ups, it's fixed now! :)